elyra/metadata/metadata_app.py

```diff
@@ -288,7 +288,7 @@ def __init__(self, **kwargs):
         super(Install, self).__init__(**kwargs)
 
 
-class ElyraMetadataApp(AppBase):
+class MetadataApplication(AppBase):
```

Comment:
This should be `MetadataApp`.  This is a convention throughout the Jupyter stack and `Application` is just too long.  This is being updated in my review changes.